src/gui/Src/BasicView/Disassembly.cpp

```diff
@@ -1928,7 +1928,7 @@ QString Disassembly::getAddrText(dsint cur_addr, char label[MAX_LABEL_SIZE])
         *label_ = 0;
     if(label)
         strcpy_s(label, MAX_LABEL_SIZE, label_);
-    return addrText;
+    return std::move(addrText);

```

Comment:
The difference is probably negligible for a small QString, you are right.
It just caught my eye and I was just wondering if there was a legitimate reason for the move.